.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Workflow - Pull Request
+
+on: [pull_request]
+
+jobs:
+  submodules:
+    name: Set up submodules
+    uses: ./.github/workflows/submodules.yml
+
+  dump:
+    name: Dump katana state
+    uses: ./.github/workflows/dump.yml
+    needs: submodules
+
+  spell_check:
+    name: Spell check
+    uses: ./.github/workflows/spell_check.yml
+
+  linters:
+    name: Linters
+    uses: ./.github/workflows/linters.yml
+
+  tests:
+    name: Rust tests
+    uses: ./.github/workflows/test.yml
+    needs: dump
+
+  # TODO remove
```

Comment:
remove ?

Reply:
Yes will remove, was testing to see production of artifact